js/build-directory.js

```diff
@@ -240,7 +240,7 @@ DataDirectory.prototype.initialiseHandlebars = function() {
   Handlebars.registerHelper('entry_thumbnail_bg', function(entry) {
     var thumbnail = entry[_this.config.thumbnail_field];
 
-    if (!thumbnail) {
+    if ((!thumbnail || !/^(http|https):\/\//.test(thumbnail)) && !Array.isArray(thumbnail)) {
```

Comment:
@hcarneiro what about URLs on native devices? They all still use `http` right?

One more thing, you can improve your regexp to:

```
/^https?:\/\//.test
```

Reply:
(the `s?` means `s` is an optional match)

Reply:
@squallstar I'll improve that. and yes, even on devices the image will always be from web URL.